main.css

```diff
@@ -90,6 +90,10 @@ blockquote footer::before {
   right: 0;
 }
 
+.v-image__image--contain {
+  background-size: 100% 100%;

```

Comment:
This stretches the pic to the ratio defined in the `v-img`. Of course, if the poster size is too far off the ratio, it would look ugly. A slight stretch looks fine.